stable_baselines3/common/vec_env/dummy_vec_env.py

```diff
@@ -73,10 +73,13 @@ def step_wait(self) -> VecEnvStepReturn:
 
     def reset(self) -> VecEnvObs:
         for env_idx in range(self.num_envs):
-            obs, self.reset_infos[env_idx] = self.envs[env_idx].reset(seed=self._seeds[env_idx])
+            obs, self.reset_infos[env_idx] = self.envs[env_idx].reset(
+                seed=self._seeds[env_idx], options=self._options[env_idx]
```

Comment:
Not sure, but this way would need `reset` to have an `options` argument while `kwargs` could be more appropriate.
but which `reset` is called?

Reply:
>  argument while kwargs could be more appropriate.

yes, for options you will need to use kwargs which should be an empty dict most of the time (so it is not a hard requirement).

>  but which reset is called? I guess it is the env_fns from self.envs so probably the true env reset.

yes, the env reset is called.